Apps/Patient/src/Services/IPatientDataService.cs

```diff
@@ -162,6 +162,35 @@ public record DiagnosticImagingExam : PatientData
         public override string Type { get; } = nameof(DiagnosticImagingExam);
     }
 
+    /// <summary>
+    /// Cancer screening exam patient data.
+    /// </summary>
+    public record CancerScreeningExam : PatientData
+    {
+        /// <summary>
+        /// Gets or sets the cancer screening's event type.
+        /// </summary>
+        public CancerScreeningType CancerScreeningType { get; set; }
+
+        /// <summary>
+        /// Gets or sets the cancer screening's program name.
+        /// </summary>
+        public string? ProgramName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the cancer screening's date.
+        /// </summary>
+        public DateTime EventTimestampUtc { get; set; }
+
+        /// <summary>
+        /// Gets or sets the cancer screening's result timestamp.
+        /// </summary>
+        public DateTime ResultTimestamp { get; set; }
```

Comment:
I'd suggest updating the documentation for all properties from "Gets or sets the cancer screening's X" to just "Gets or sets the X".